src/linear-algebra.lisp

```diff
@@ -249,7 +249,7 @@ The function will just return NIL, and modify the contents of RESULT."
   (:method ((gate quil:simple-gate))
     (make-instance 'quil:simple-gate
                    :name (concatenate 'string (quil:gate-name gate) "*")
-                   :matrix (magicl:conjugate-entrywise (quil:gate-matrix gate))))
+                   :matrix (magicl:map! #'conjugate (quil:gate-matrix gate))))
```

Comment:
Does the `!` indicate mutation? I don't know if it matters here, but the old `conjugate-entrywise` did not mutate the input.

Reply:
@colescott what about this one? ok to `map!`?

Reply:
@stylewarning

Reply:
No we very much shall not mutate. You're destroying the environment aka the original gate binding matrix.